.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,49 @@
+name: "Bug Report (Low Priority)"
+description: "Create a public Bug Report. Note that these may not be addressed as quickly as the helpdesk and that looking up account information will be difficult."
+title: "[BUG]: "
+labels: bug
+body:
+  - type: input
+    attributes:
+      label: Tracer Version(s)
+      description: "Version(s) of the tracer affected by this bug"
+      placeholder: 1.70.0
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Go Version(s)
+      description: "Version(s) of Go (`go version`) that you've encountered this bug with. If you aren't using the [latest version](https://github.com/DataDog/dd-trace-go/releases) of dd-trace-go, try upgrading first to see if your issue has already been resolved."
```

Comment:
Are we asking for the version of _Golang_ that they're using, or the version of the _Go Tracer_? From reading the label and placeholder value, it seems like Golang, but the description implies otherwise. 

I would find the version of the Go Tracer more useful. Could we change the label to say "Go Tracer Version(s)" and the placeholder to say something like "v1.71.0"?